src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1219,9 +1217,24 @@ int fromCommandLine (
         nRet                    = rpcINTERNAL;
     }
 
-    std::cout << jvOutput.toStyledString ();
+    return { nRet, std::move(jvOutput) };
+}
+
+//------------------------------------------------------------------------------
+
+namespace RPCCall {
+
+int fromCommandLine (
+    Config const& config,
+    const std::vector<std::string>& vCmd,
+    Logs& logs)
+{
+    auto const result = rpcClient(vCmd, config, logs);
+
+    if (result.first != 1)
```

Comment:
Instead of `EXIT_...`, how about `rpcSUCCESS` and `rpcBAD_SYNTAX`?
https://github.com/ximinez/rippled/blob/mao/src/ripple/protocol/ErrorCodes.h#L34-L36

`rpcClient` already returns `rpcINTERNAL` at https://github.com/ximinez/rippled/blob/mao/src/ripple/net/impl/RPCCall.cpp#L1217